TCAT/Controllers/HomeOptionsCardViewController.swift

```diff
@@ -428,11 +426,11 @@ extension HomeOptionsCardViewController: HomeMapViewDelegate {
         switch connection {
         case .none:
             isNetworkDown = true
-            searchBar.isUserInteractionEnabled = false
+//            searchBar.isUserInteractionEnabled = false
```

Comment:
delete?

Reply:
Good catch. Didn't mean to comment this actually. Ima double check the whole pr to make sure stuff like this didn't happen elsewhere.